ofrak_core/CHANGELOG.md

```diff
@@ -6,9 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
 ### Fixed
 - Fix long-broken `OFRAK.set_id_service` [#198](https://github.com/redballoonsecurity/ofrak/pull/198)
+- Delete narrower list of resources when using `SegmentInjectorModifier` [#200](https://github.com/redballoonsecurity/ofrak/pull/200)
 
 ### Changed
 - GUI is much faster, especially for resources with hundreds of thousands of children [#191](https://github.com/redballoonsecurity/ofrak/pull/191)
+- Resources whose data gets modified are now listed in the `resource_modified` field of component results [#200](https://github.com/redballoonsecurity/ofrak/pull/200)
```

Comment:
I don't understand this. Does this mean that previously only resource analysis were listed in resource_modified? I think this change could be clearer

Reply:
Previously only changes to the resource _model_ caused a resource to be add to this "modified" set. Data patches could trigger that indirectly (that is, a data patch invalidates a some attributes that resource has which depends on that data, which is a change to the model) but were not guaranteed to do so. So the change is as it says: Resources whose data gets modified are now *definitely* listed in the `resource_modified` field of component results.